sample/format.hpp

```diff
@@ -579,6 +579,14 @@ inline std::ostream& operator<<(std::ostream& out,
     return out;
 }
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const pfs::uptime& system_uptime)
```

Comment:
You can just use "uptime" for the variable since the type requires the "pfs" prefix.

Reply:
Done.